app/models/base_item.rb

```diff
@@ -28,5 +28,24 @@ class BaseItem < ApplicationRecord
   def to_h
     { partner_key: partner_key, name: name }
   end
-end
 
+  def self.seed_base_items
```

Comment:
This is only used in specs and seeds. Not sure this belongs on the main BaseItem class. I'd probably create a module call `Seeds` or something like that that can be referenced from those two contexts.

Reply:
moved into `lib/seed_base_items.rb` in PR #4665 [8fde086](https://github.com/rubyforgood/human-essentials/pull/4665/commits/8fde086ec1cdf4e3a346ec452e43406113265e02)